pubsublite/spark-connector/README.md

```diff
@@ -0,0 +1,185 @@
+   export BUCKET_ID=your-gcs-bucket-id
+
+   gsutil mb gs://$BUCKET_ID
+   ```
+
+## Python setup
+
+1. [Install Python and virtualenv].
+
+1. Clone the `python-docs-samples` repository.
+
+    ```bash
+    git clone https://github.com/GoogleCloudPlatform/python-docs-samples.git
+    ```
+
+1. Navigate to the sample code directory.
+
+    ```bash
+    cd python-docs-samples/pubsublite/spark-connector
+    ```
+
+1. Create a virtual environment and activate it.
+
+    ```bash
+    virtualenv env
+    source env/bin/activate
+    ```
+   > Once you are finished with the tutorial, you can deactivate
+   > the virtualenv and go back to your global Python environment
+   > by running `deactivate`.
+
+1. Install the required packages.
+    ```bash
+    python -m pip install -U -r requirements.txt
+    ```
+
+## Creating a Spark cluster on Dataproc
+
+1. Go to [Cloud Console for Dataproc].
+
+1. Go to Clusters, then [Create Cluster].
+   > **Note:** When setting up the cluster, you must choose
+   > [Dataproc Image Version 1.5] under ___Versioning___ because
+   > the connector currently only supports Spark 2.4.8.
+   > Additionally, in ___Manage security (optional)___, you
+   > must enable the cloud-platform scope for your cluster by
+   > checking "Allow API access to all Google Cloud services in
+   > the same project" under ___Project access___.
+
+   Here is an equivalent example using a `gcloud` command, with an additional optional argument to enable component gateway:
+
+    ```sh
+    export CLUSTER_ID=your-cluster-id
+    export CLOUD_REGION=your-region
+
+    gcloud dataproc clusters create $CLUSTER_ID \
+      --region $CLOUD_REGION \
+      --image-version 1.5-debian10 \
+      --scopes 'https://www.googleapis.com/auth/cloud-platform' \
+      --enable-component-gateway
+    ```
+
+## Writing to Pub/Sub Lite
+
+[spark_streaming_to_pubsublite_example.py](spark_streaming_to_pubsublite_example.py) creates a streaming source of consecutive numbers with timestamps for 60 seconds and writes them to a Pub/Sub topic.
+
+To submit a write job:
+
+```sh
+export PROJECT_NUMBER=$(gcloud projects list --filter="projectId:$PROJECT_ID" --format="value(PROJECT_NUMBER)")
+
+gcloud dataproc jobs submit pyspark spark_streaming_to_pubsublite_example.py \
+    --region=$CLOUD_REGION \
+    --cluster=$CLUSTER_ID \
+    --jars=https://search.maven.org/remotecontent?filepath=com/google/cloud/pubsublite-spark-sql-streaming/0.3.1/pubsublite-spark-sql-streaming-0.3.1-with-dependencies.jar \
+    --driver-log-levels=root=INFO \
+    --properties=spark.master=yarn \
+    -- $PROJECT_NUMBER $LOCATION $TOPIC_ID
+```
+
+The preceding command hardcodes the connector version. Visit this Maven [link](https://search.maven.org/artifact/com.google.cloud/pubsublite-spark-sql-streaming) to learn and download the latest version of the connector's uber jar.
+
+Visit the job URL in the command output or the jobs panel in [Cloud Console for Dataproc] to monitor the job progress.
```

Comment:
Could you provide an example of what the output would look, similar to how you did below with reading from Pub/Sub Lite?